src/engine/BMGameAction.php

```diff
@@ -571,6 +571,17 @@ protected function friendly_message_determine_initiative() {
         return $message;
     }
 
+    protected function friendly_message_play_another_turn() {
+        $message = $this->outputPlayerIdNames[$this->actingPlayerId] .
+                   ' gets another turn';
+
+        if ('TimeAndSpace' == $this->params['cause']) {
+            $message .= ' because a TimeAndSpace die rolled odd';
```

Comment:
IMO, even if the internal skill name is TimeAndSpace (which i'm fine with) the message should say "Time and Space"